.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDurationExtractor.cs

```diff
@@ -34,13 +34,14 @@ public List<ExtractResult> Extract(string text, DateObject reference)
 
             var rets = Token.MergeAllTokens(tokens, text, ExtractorName);
 
-            ResolveMoreThanOrLessThanPrefix(text, rets);
-
+            // First MergeMultipleDuration then ResolveMoreThanOrLessThanPrefix so cases like "more than 4 days and less than 1 week" will not be merged into one "multipleDuration"
             if (this.merge)
             {
                 rets = MergeMultipleDuration(rets, text);
             }
 
+            ResolveMoreThanOrLessThanPrefix(text, rets);

```

Comment:
What does "Resolve" means in this case? Maybe just call this method by a better name?